src/webob/acceptparse.py

```diff
@@ -255,11 +294,382 @@ class AcceptEncoding(Accept):
     """
 
 
-class AcceptLanguage(Accept):
+class AcceptLanguage(object):
     """
-    Represents an ``Accept-Language`` header.
+    Represent an ``Accept-Language`` header.
+
+    Base class for :class:`AcceptLanguageValidHeader`,
+    :class:`AcceptLanguageNoHeader`, and :class:`AcceptLanguageInvalidHeader`.
     """
-    def _match(self, mask, item):
+
+    @classmethod
+    def python_value_to_header_str(cls, value):
```

Comment:
But `.parse()` is documented api that (I assume) we can't break? It currently returns an iterator, so I have kept that the same in my PR.

In case it's not clear, `.python_value_to_header_str()` is not intended to be public API &mdash; it's not documented, and is just a helper method. I only didn't prefix the name with an underscore because it needs to be called from `accept_language_property` too (but maybe I should?) So there's only one API to create a header from a non-`str` Python value: `AcceptNoHeader() + ...`.